admin/includes/languages/english/geo_zones.php

```diff
@@ -1,54 +1,23 @@
 <?php
 /**
  * @package admin
- * @copyright Copyright 2003-2006 Zen Cart Development Team
+ * @copyright Copyright 2003-2015 Zen Cart Development Team
  * @copyright Portions Copyright 2003 osCommerce
  * @license http://www.zen-cart.com/license/2_0.txt GNU Public License V2.0
- * @version $Id: geo_zones.php 4736 2006-10-13 07:11:44Z drbyte $
+ * @version $Id:  Modified in v1.6.0 $
  */
 
 define('HEADING_TITLE', 'Zone Definitions - Taxes, Payment and Shipping');
-
-define('TABLE_HEADING_COUNTRY', 'Country');
-define('TABLE_HEADING_COUNTRY_ZONE', 'Zone');
-define('TABLE_HEADING_TAX_ZONES', 'Zone Name');
-define('TABLE_HEADING_TAX_ZONES_DESCRIPTION', 'Zone Description');
-define('TABLE_HEADING_STATUS', 'Status');
-define('TABLE_HEADING_ACTION', 'Action');
-//define('TEXT_LEGEND', 'LEGEND: ');
+define('TEXT_ENTRY_COUNTRY', 'Country');
+define('TEXT_ENTRY_COUNTRY_ZONE', 'Zone');
+define('TEXT_ENTRY_TAX_ZONES', 'Zone Name');
+define('TEXT_ENTRY_TAX_ZONES_DESCRIPTION', 'Zone Description');
+define('TEXT_ENTRY_GEO_ZONES_STATUS', 'Status');
+define('TEXT_ENTRY_ACTION', 'Action');
+define('TEXT_LINK_DETAILS', 'Details');

```

Comment:
Can we change the "Details" link to read "Edit Sub-Zone"

And maybe change the "Edit" link to read "Edit Name". Granted, the "Edit" text probably comes from a generic lang-def and not from this same language file ...?